reducers/images_reducer.js

```diff
@@ -121,6 +122,18 @@ export default function(state = INITIAL_STATE, action) {
 
                 break;
 
+            /**
+             * Changes litter picked up status of all images
+             * to payload
+             *
+             * action.payload -- {boolean}
+             */
+            case CHANGE_LITTER_STATUS:
+                console.log(action.payload);
+                draft.imagesArray.map(img => (img.picked_up = action.payload));
```

Comment:
`()` can be removed here, it looks like its evaluating something which is not the case